customize/packer.lua

```diff
@@ -201,6 +201,12 @@ local function install_plugins(plugins, lr_flag)
   end
 end
 
+local function create_plugins_directory() 
+  local ok = exec("mkdir plugins")
```

Comment:
Better do this to stay on the safe side:

```suggestion
  local ok = exec("mkdir -p /plugins")
```